client/components/domains/domain-product-price/index.jsx

```diff
@@ -6,7 +6,10 @@ import React from 'react';
 /**
  * Internal dependencies
  */
-import PremiumPopover from 'components/plans/premium-popover';
+const PremiumPopover = require( 'components/plans/premium-popover' ),
```

Comment:
Why did you switch to the "old" style of importing modules (via `require`)? The external dependencies are imported via `import` so I think we should keep it consistent.